app/src/main/assets/chart.html

```diff
@@ -2,11 +2,7 @@
 <link rel="stylesheet" href="chart.css">
```

Comment:
To add more considerations on the matter of the <!DOCTYPE html> and <html> tags here is a screenshot I made:
![screenshot from 2015-10-06 01 58 14](https://cloud.githubusercontent.com/assets/1710823/10301119/abaf4fdc-6bd5-11e5-8e3e-75f018ded875.png)

As you can see, the browser added the html tag even though it wasn't present in the html file. This doesn't mean that we shouldn't use the html tag. 
Explicitly declaring html5 document would avoid odd behaviors when using most advanced html features. This is to tell exactly how the browser (webview) should render the html.
Here are some info:
http://www.w3.org/QA/2002/04/valid-dtd-list.html
http://www.w3.org/wiki/Choosing_the_right_doctype_for_your_HTML_documents
https://hsivonen.fi/doctype/
http://alistapart.com/article/doctype
http://www.w3schools.com/tags/tag_doctype.asp